### PR TITLE
Adding migrateIds config

### PR DESCRIPTION
**Pull request by jileeshs**
Adding `migrateIds` config

## Description

Add `migrateIds` config

## Related Issue
https://github.com/adobe/reactor-extension-alloy/tree/feature/CORE-37383

## Screenshots (if appropriate):
Screen Shot 2019-11-14 at 11 21 03 AM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All tests pass and I've made any necessary test changes.
- [x] I've updated the schema in extension.json or no changes are necessary.

### CODE REVIEW

File: src/view/configuration/configuration.jsx [hunk 1]
```diff
@@ -55,7 +55,8 @@ const getInstanceDefaults = initInfo => ({
   cookieDestinationsEnabled: true,
   prehidingStyle: "",
   contextGranularity: contextGranularityEnum.ALL,
-  context: contextOptions
+  context: contextOptions,
+  migrateIds: false
 });
```
Comment (jonsnyder) [inline src/view/configuration/configuration.jsx]: We talked a while back about merging the alloy and alloy-extension code bases.  Looking at this instanceDefaults object, there is an opportunity here to run an automated test to make sure these stay in sync with the validators in alloy.

File: src/view/configuration/configuration.jsx [hunk 2]
```diff
@@ -474,6 +476,16 @@ const Configuration = () => {
                             </div>
                           ) : null}
 
+                            <div className="u-gapTop">
+                            <InfoTipLayout tip="Copy id from a previous version of experience cloud ID service.">
```
Comment (Aaronius) [inline src/view/configuration/configuration.jsx]: "id" should probably be "ID". I'm guessing "experience cloud ID service" should be "Adobe Experience Cloud Identity Service" for branding/legal reasons.

File: src/view/configuration/configuration.jsx [hunk 3]
```diff
@@ -474,6 +476,16 @@ const Configuration = () => {
...
+                              <WrappedField
+                                name={`instances.${index}.migrateIds`}
+                                component={Checkbox}
+                                label="Migrate Ids from VisitorAPI"
```
Comment (Aaronius) [inline src/view/configuration/configuration.jsx]: "Ids" => "IDs" or "identities"

**Comment by Aaronius** [comment]
@jileeshs it looks like tests still need to be updated. See https://github.com/adobe/reactor-extension-alloy/pull/35#pullrequestreview-315781211

**Comment by Aaronius** [comment]
Thanks!

**Pull request merged**